code/game/objects/effects/overlays.dm

```diff
@@ -494,9 +496,9 @@
 	icon = 'icons/mob/mob.dmi'
 	duration = 15
 
-/obj/effect/overlay/temp/gib_animation/New(loc, gib_icon)
+/obj/effect/overlay/temp/gib_animation/Initialize(mapload, gib_icon)
 	icon_state = gib_icon // Needs to be before ..() so icon is correct
-	..()
```

Comment:
See comment

This was running code before parent New() that would now happen after New().

Would the behavior be correct now that its running that code after new but before initialize?